src/resources/schema/social-profile.json

```diff
@@ -0,0 +1,31 @@
+{
+  "$id": "social-profile.json",
+  "title": "Social Profile",
+  "type": "object",
+  "description": "All social profile",
+  "properties": {
+    "name": {
```

Comment:
Ideally, one could think of a social profile as a standalone collection where a social profile is shared by multiple projects. An example could be Routescan and Avascan.

Reply:
Oh I see, so you can have multiple social profiles?

Take a look at the blockchain-address.json schema. Would that approach make sense here?

Namely, we could have a social-profile schema that stores `{ name, handle, url, type }`. The `type` is an enum which can be one of {discord, telegram, twitter}. Then we just store a bunch of social profiles in an array in the project.

Reply:
probably if we use this approach it could be expensive in terms of data. I reverted the previous commit, and I use the social schema with array of urls to include multiple groups/channels for every key